docs/Conformance-Guide-draft.md

```diff
@@ -0,0 +1,103 @@
+# Proposal for Zowe Explorer Conformance Criteria for Zowe Conformance v1
+
+**Note, this is a draft for potential future conformance criteria for extending Zowe Explorer for collecting feedback, only**
+
+## Kinds of extensions
+
+The following kinds of extensions can be certified for Zowe Explorer. One extension offering needs to comply with at least one of these kinds, but it can also be in several or all of the kinds listed. In addition to fulfilling all the Required criteria for at least one kind (1 to 3), the candidate extension also needs to fulfill the required criteria of the (0) General category.
+
+0. **General** extension.
+1. **Profiles access**: An extension that accesses the Zowe Explorer Zowe CLI profiles caches for read or write operations.
+2. **Data Provider**: An extension that provides an alternative Zowe CLI profile type for a different z/OS communication protocol for one or more Zowe Explorer views.
+3. **Menus**: An extension that injects new menus into the existing Zowe Explorer tree views accessing contextual information from the nodes.
+
+## Format for conformance criteria
+
+Every conformance criteria needs to be clearly scoped, defined, and objectively verifiable. We therefore specify the following in each of the criteria descriptions below:
+
+- Unique name: each criteria has a short unique name that can be used for reference and discussion amongst parties
+- Required vs Best Practice: we currently distinguish if a criteria is a must-do for conformance versus if it is just a recommended best practice.
+- Goal: each criteria should in its description clearly state the goal or rationale for requiring it.
+- Testability: Each criteria should specify the way and extender as well as the Zowe Explorer squad can verify conformance.
+
+## General conformance criteria (0)
+
+These criteria are independent of the kind of extension produce and need to be fulfill for all cases.
+
+1. (Required) Naming: **If the extension uses the word "Zowe" in its name, it abides by Linux Foundation's Trademark Usage rules to ensure the word Zowe is used in a way intended by the Zowe community.**
+
+   The extender needs to provide a link where the offering will be announced and/or made available to the public.
+
+1. (Required): No Zowe CLI plugin installation requirement: **If the extender makes use of a Zowe CLI profile other than the Zowe Explorer default `zosmf` then the extension must not make any assumptions that a matching Zowe CLI plugin has been installed in the Zowe Explorer user's environment.**
```

Comment:
**No Zowe CLI plugin installation dependency:** If the extension makes use of a Zowe CLI profile other than the Zowe Explorer default `zosmf`, it must not make any assumptions that a matching Zowe CLI plugin has been installed in the Zowe Explorer user's environment.